src/plugins/point_in_time_management/public/components/empty_state/empty_state.tsx

```diff
@@ -0,0 +1,70 @@
+/*
+ * Copyright OpenSearch Contributors
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+import React from 'react';
+import { FormattedMessage } from '@osd/i18n/react';
+import {
+  EuiPageContentHeader,
+  EuiPageContentHeaderSection,
+  EuiTitle,
+  EuiPageContent,
+  EuiSpacer,
+  EuiText,
+  EuiPageContentBody,
+  EuiFlexItem,
+  EuiFlexGroup,
+  EuiButton,
+} from '@elastic/eui';
+
+export const EmptyState = () => {
+  return (
+    <>
+      <EuiPageContent
+        className="pitEmptyState"
+        grow={false}
+        style={{ minHeight: '70vh' }}
+        horizontalPosition="center"
+        data-test-subj="pointInTimeEmptyState"
+      >
+        <EuiPageContentHeader>
+          <EuiPageContentHeaderSection>
+            <EuiTitle>
+              <h1>
+                <FormattedMessage
+                  id="pointInTimeManagement.pointInTime.header.pointInTimeTitle"

```

Comment:
why `pointInTimeManagement.pointInTime` instead of just `pointInTimeManagement`?